tox.ini

```diff
@@ -4,10 +4,9 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27-test, py37-test, py27-flake8, docs
-
-# The exhaustive list of environments is:
-# envlist = py{27,34,35}-{test,cov}, py{27,34,35}-flake8, docs
+envlist =
+    py{27,35,36,37,38}-{test,cov,flake8}
+    docs
```

Comment:
The reason we had this pared down to a smaller list is because it controls what happens by default when you just type `tox` at the command line. It's pretty annoying to have this launch all 5*3=15 versions in the common case, so we narrowed this down to a smaller list that is good for most development. The shorter list does not prevent CI from using whatever envs it wants.

Reply:
Ah, that makes sense. So all these environment's on this list are automatically ran when you run `tox` I assume?

Reply:
Yep!